js/source/raster_tile_source.js

```diff
@@ -37,15 +40,27 @@ class RasterTileSource extends Evented {
         this.map = map;
     }
 
+    setBounds(bounds) {
+      this.bounds = bounds;
+      if (bounds) {
+        this.tileBounds = new TileBounds(bounds, this.minzoom, this.maxzoom);
+      }
+    }
+
     serialize() {
         return {
             type: 'raster',
             url: this.url,
             tileSize: this.tileSize,
-            tiles: this.tiles
+            tiles: this.tiles,
+            bounds: this.bounds,
```

Comment:
`serialize` is expected to return a plain object (aka JSON serializable object, POJO). `this.bounds` is an instance of `TileBounds`. We'll need to think of a way to serialize `this.bounds` into a plain object, probably by writing a `TileBounds#serialize` method.